db/migrate/20200203190734_add_funder_and_org_to_plans.rb

```diff
@@ -0,0 +1,6 @@
+class AddFunderAndOrgToPlans < ActiveRecord::Migration
+  def change
+    add_reference :plans, :org, foreign_key: true
+    add_column :plans, :funder_id, :integer, index: true
```

Comment:
Does this point out at a  ROR funder id?
If so, could we get rid of the `funder_name` field on the plan and just use the `name` of the funder with id `funder_id`?

Reply:
yes. There is data in that funder_name field (at least in DMPTool) and I had meant to write a script to try and convert it over to an actual funder. I saw the funder_name cleanup as part of a second pass where we truly eliminate the org/user_identifiers tables and remove the is_other flag and default Org